aries-site/src/examples/components/dateinput/DateInputValidationExample.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Box, Form, FormField, DateInput, Text } from 'grommet';
+import PropTypes from 'prop-types';
+import { CircleAlert } from 'grommet-icons';
+
+const Error = ({ children, ...rest }) => {
+  return (
+    <Box direction="row" {...rest}>
+      <Box flex={false} margin={{ top: 'xxsmall', right: 'xxsmall' }}>
+        <CircleAlert size="small" />
+      </Box>
+      <Text size="xsmall">{children}</Text>
+    </Box>
+  );
+};
+
+Error.propTypes = {
+  children: PropTypes.string,
+};
+
+export const DateInputValidationExample = () => {
+  const [value, setValue] = React.useState('');
+  const [message, setMessage] = React.useState(
+    <Error background="background-front">Select date to resolve error</Error>,
+  );
+  const onChange = event => {
+    const nextValue = event.value;
+    console.log('onChange', nextValue);
+    setValue(nextValue);
+    setMessage('');
```

Comment:
This looks like we always set the message to ''. What if I remove the value? Shouldn't it go back to 'required'?

Reply:
I can look at doing that with removing the value then for it to go back to required

Reply:
looks like `onChange` is not getting called when there is no date so I will open an issue on grommet